packages/eslint-plugin-js/rules/lines-around-comment/lines-around-comment.ts

```diff
@@ -3,7 +3,11 @@
  * @author Jamund Ferguson
  */
 
+import { AST_NODE_TYPES } from '@typescript-eslint/utils'
```

Comment:
We need to avoid importing it **at runtime**, as we only deps `@typescript-eslint/utils` on developement.